tests/test_filtsmooth/test_gaussfiltsmooth/test_iterated_kalman.py

```diff
@@ -1,14 +1,13 @@
 import numpy as np
 import pytest
 
+import probnum.problems.zoo.filtsmooth as pn_filtsmooth_zoo
```

Comment:
do you think the `pn_` adds information? I am not sure, but this sort of import style could be something that we want to adopt package-wide, so let's do this as perfectly as possible from the beginning on.